### PR TITLE
DMEERX-600 Home Blood Glucose F2F Encounter Questionnaire

### PR DESCRIPTION
**Pull request by yuliumitre**
This PR creates the F2F Encounter Questionnaire for Home Blood Glucose Monitor DME.

To test this PR, switch to the bgm-dtr branch of test-ehr repo. https://github.com/HL7-DaVinci/test-ehr/pull/34

There are several tests based on the device request orders and the diagnosis of the patient.
1. Test Non-CGM
Patient: pat014 
Device request : E0607
Verify "Treatment Plan" Section
2. Test CGM
Patient: pat014
Device request: S1031
Verify "Treatment Plan" Section
3. Test patient diabetes diagnosis has been confirmed
Patient: pat014
Verify: 
1) Verify the "Diabetes Mellitus" under Patient Diagnosis is prepopulated
2) Under "Lab validation" section, "Confirming diagnosis of Diabetes Mellitus" is not checked
4. Test patient has not existing diabetes diagnosis in EHR
Patient: pat016
Verify:
1) Verify the "Diabetes Mellitus" under Patient Diagnosis is not prepopulated
2) Under "Lab validation" section, "Confirming diagnosis of Diabetes Mellitus" is checked

### CODE REVIEW

**Comment by lpdecelles** [comment]
Yolonda, I was not able launch DTR. I used the bgm-dtr branches from both test-ehr and CDS-Library. Are you able to launch it?
image

**Comment by yuliumitre** [comment]
Yes, @lpdecelles Larry, I am able to load the questionnaire. I am suspecting the ValueSets haven't been reloaded. Can you reload your valueset as I added couple more? Thanks.

File: HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorFaceToFace.json [hunk 1]
```diff
@@ -0,0 +1,506 @@
...
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"HomeBloodGlucoseMonitorFaceToFacePrepopulation\".DMConfirmedDate"
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
```
Comment (yunwwang) [inline HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorFaceToFace.json:170]: 6,7,8,9 can be replaced by Subject sub-questionnaire. That sub-q does not use the CQL expression AllProcedure. You can add that to sub-q

File: HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorFaceToFace.json [hunk 2]
```diff
@@ -0,0 +1,506 @@
...
+      "extension": [
+        {
+          "url": "http://hl7.org/fhir/StructureDefinition/sub-questionnaire",
+          "valueCanonical": "questionnaire/review-of-system"
+        }
+      ],
+      "type": "display"
+    },
+    {
+      "linkId": "11",
```
Comment (yunwwang) [inline HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorFaceToFace.json:234]: PhysicanExam is split into two sub-qs: VitalSigns and PE. You may need to wait for ROS branch to be merged

**Comment by lpdecelles** [comment]
I will try it again now.

**Comment by lpdecelles** [comment]
It works now. I needed to reload the value sets. I have opened a separate Jira ticket to address the value set issue.

**Comment by lpdecelles** [comment]
@yunwwang can we merge this and add the sub-questionnaires at a later date?

**Comment by yuliumitre** [comment]
Yeah, sub-questionnaire should be another story :-)

File: HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorFaceToFace.json [hunk 3]
```diff
@@ -0,0 +1,506 @@
...
+      "extension": [
+        {
+          "url": "http://hl7.org/fhir/StructureDefinition/sub-questionnaire",
+          "valueCanonical": "questionnaire/practitioner-info"
+        }
+      ],
+      "type": "display"
+    },
+    {
+      "linkId": "3",
```
Comment (yunwwang) [inline HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorFaceToFace.json:44]: ROS branch has a new subQuestionnaire EncounterItems which can be used here.

**Pull request merged**